src/mds/MDBalancer.cc

```diff
@@ -37,6 +37,10 @@ using namespace std;
 #include "common/config.h"
 #include "common/errno.h"
 
+/* Note, by default ceph_mds_balancer is 1/5. For debug messages 1<lvl<=5, should_gather will
+ * be true so whatever debug_mds is is ignored. So, some messages may not appear in the log.
```

Comment:
".....be true so whatever debug_mds is, it is ignored. So, some messages may not appear in the log."

Just an opinion: s/is is/is, it is